tailwind.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  content: ["./app/**/*.{ts,tsx,jsx,js}"],
+  theme: {
+    color: {
+      resolutionBlue:	'#041484',
+      razzmatazz: '#E30464',
+      feijoa: '#96D882',
+      fern:	'#6DB470'
```

Comment:
Can we have 1 tab spacing here as above please ??

Reply:
resolved